src/swarm-engine/Swarm/Game/Step/Const.hs

```diff
@@ -880,10 +880,14 @@ execConst runChildProg c vs s k = do
       [VText app, VInj hasAttr mattr] -> do
         -- Set the robot's display character(s)
         case into @String app of
+          [] -> do
+            robotDisplay . invisible .= True
```

Comment:
@byorgey what do you think? I could limit it to system robots if it is too powerful, but since it requires `beaglepuss` it sounds plausible to me. 🥸

Reply:
Sure, I like it, why not?

We may need to reconsider if we ever get some kind of adversarial multiplayer mode (i.e. playing against an opponent in a shared world over the network).  A `beaglepuss` is not that hard to make (`rubber` + `glass` + `string`) so this would be pretty OP if you could do stuff to your opponent with invisible robots.  But for now it's fun and doesn't really matter.